lib/galaxy/datatypes/binary.py

```diff
@@ -657,7 +657,8 @@ def set_meta(self, dataset, overwrite=True, **kwd):
                                        '__dataset_%d_%s' % (dataset.id, os.path.basename(index_file.file_name)))
         os.symlink(dataset.file_name, dataset_symlink)
         try:
-            pysam.bcftools.index(dataset_symlink)
+            cmd = ['python', '-c', "import pysam.bcftools; pysam.bcftools.index('%s')" % (dataset_symlink)]
```

Comment:
Can you add a comment along the lines of the message for commit cc177b550f7d04c2d40c6ec16d82115bf86d60fc ?